bin/build.sh

```diff
@@ -331,6 +331,7 @@ then
   mozroots --import --sync
   ${WGET} https://nuget.org/nuget.exe
   mono ./nuget.exe restore Boogie.sln
+  rm /tmp/nuget/ -rf 
```

Comment:
Please update this into rm -rf /tmp/nuget/ to make it consistent with other rm commands.